### PR TITLE
ci: Update 1% payload size test

### PR DESCRIPTION
**Pull request by tinayuangao**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ x] CI related changes
[ ] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?
Current 1% payload size check fetch data from firebase, when the change exceed 1% there's no way to fix/approve it.

## What is the new behavior?
Save payload size data locally and when change > 1%, we can update the `payload_limits.log` file to change the payload size limits. 

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: scripts/ci/payload-size.js [hunk 1]
```diff
@@ -1,21 +1,32 @@
 
 const fs = require('fs');
-const latest = JSON.parse(fs.readFileSync('/tmp/latest.log', 'utf8'));
+// Get branch and project name from command line arguments
+const limitFile = process.argv[2];
+const project = process.argv[3];
+const branch = process.argv[4];
```
Comment (rafaelss95) [inline scripts/ci/payload-size.js]: What about `const [, , limitFile, project, branch] = process.argv[`? :)

**Review by gkalpak — COMMENTED** [review]
Renaming `_payload-limits.sh` to `_payload-limits.log` while keeping them executable is unexpected. Since the files are consumed as JSON, they should probably be converted to JSON (and renamed to `.json`) :grin:

**Comment by vicb** [comment]
question: is there a place where we can see a graph of size vs time out of firebase ?

**Comment by tinayuangao** [comment]
You can view the graph here: https://angular-payload-size.firebaseapp.com/

**Review by mhevery — APPROVED** [review]

**Comment by mhevery** [comment]
please add merge and target as needed.

**Comment by IgorMinar** [comment]
@tinayuangao  please add the correct target label in the future. in general for docs, bug fixes and infrastructure changes we should always try to cherrypick the commit to the stable branch.

**Pull request closed without merging**